test/unit/specs/components/AudioTrack/v-audio-track.spec.js

```diff
@@ -30,7 +30,10 @@ jest.mock('~/composables/use-browser-detection', () => ({
 }))
 
 jest.mock('~/composables/use-i18n', () => ({
-  useI18n: jest.fn(() => mockI18n),
+  useI18n: jest.fn(() => ({
```

Comment:
mockI18n cannot correctly handle the nested keys for `$t`: if there are several parts of the key (`hero.title`), it tries to get the `messages['hero.title']` instead of splitting the key by '.', and getting all of them like `messages['hero']['title']`. This is why I removed it.